static/app/views/projectDetail/projectLatestAlerts.tsx

```diff
@@ -42,7 +42,7 @@ function AlertRow({alert, orgSlug}: AlertRowProps) {
       aria-label={title}
       to={`/organizations/${orgSlug}/alerts/${identifier}/`}
     >
-      <AlertBadgeWrapper {...statusProps} icon={Icon}>
+      <AlertBadgeWrapper {...statusProps} icon={<Icon />}>
```

Comment:
I think this is the only type change. Its declared a few lines up and is a component